### DISCUSSION TITLE
Disable HTTPS

### DISCUSSION THREAD

**Discussion by ibzann** [vercel/vercel#6303] [Help] [answered]
Hello, I'm wondering if I could disable HTTPS on my vercel site as Cloudflare completely breaks it with its SSL. Any way of doing this?

**Comment by ibzann** [comment, marked as answer]
fixed it

**Comment by fu-sen** [comment]
I run into this issue when using Vercel via Cloudflare. Vercel has had that experience for a long time, so there is a blog post for that.
https://vercel.com/support/articles/using-cloudflare-with-vercel
